lib/improver/cli/nowcast_extrapolate.py

```diff
@@ -100,6 +103,20 @@ def main(argv=None):
                         help="Maximum lead time required (mins).")
     parser.add_argument("--lead_time_interval", type=int, default=15,
                         help="Interval between required lead times (mins).")
+
+    accumulation_args = parser.add_argument_group(
+        'Calculate accumulations from advected fields')
+    accumulation_args.add_argument(
+        "--accumulation_fidelity", type=int, default=0,
```

Comment:
`Fidelity` is a long word - can we make this a bit more plain English / descriptive please.  `--accumulation_interval`?

Reply:
I worry that it sounds like lead_time_interval which describes the desired output interval, whereas this is the calculation interval that is used to produce accumulations over periods equal to the lead_time_interval. We can problem think of another term.

Reply:
Let me get my thesaurus...

Reply:
Going to leave this as it's been a week and I can't think of anything better.